sagemaker-triton/fil_ensemble/1_prep_rapids_train_xgb.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<p>For accessibility and clarity, please use blog/article titles for link text, rather than other text. For instance, <code>Although RAPIDS on SageMaker is excellent for running cost-effective HPO in minimal amount of time to get to the best accuracy model configuration, as shown in the blog post [RAPIDS and Amazon SageMaker: Scale up and scale out to tackle ML challenges](link.html)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Thanks for that. i have updated the text and link